bundle/src/lib/header-bidding/prebid/prebid.ts

```diff
@@ -392,9 +393,11 @@ const initialise = async (
 			email &&
 			!isUserInTestGroup('commercial-user-module-ID5', 'variant')
 		) {
-			const hashedEmail = await hashEmail(email);
+			const hashedId5Email = await hashEmail(email, 'id5');
```

Comment:
note: this const is scoped within the if block so you don't have to rename it `hashId5Email` as it won't conflict with the `hashedUid2email` below if it was called `hashedEmail` too